src/main.rs

```diff
@@ -20,25 +20,36 @@ fn main() -> Result<(), Box<dyn Error>> {
         process::exit(0);
     }
 
-    let total = warnings.len();
+    let mut total = 0;
+    warnings.iter().for_each(|w| total += w.warnings.len());
     let is_not_quiet = !args.is_present("quiet");
 
     if args.is_present("fix") {
+        // Don't display anything when there are no warnings
+        if total == 0 {
+            process::exit(0);
```

Comment:
I think you can do it earlier before `args.is_present("fix")`.

Reply:
I realized I had this in the wrong spot. To support the case where we still output "Checking FILE" even when there are no warnings at all, we'd have to do this check later instead of earlier.

In the case of no warnings, should we output text to indicate that? I.e. instead of "All warnings are fixed. Total:" we say "No warnings found".

Reply:
I like that idea 🙂